packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx

```diff
@@ -212,6 +217,7 @@ export const SelectPlanPanel: React.FC<CombinedProps> = (props) => {
             }
             aria-disabled={isSamePlan || planTooSmall || isDisabledClass}
             className={classNames({
+              [classes.focusedRow]: true,
               [classes.disabledRow]:
                 isSamePlan || planTooSmall || isDisabledClass,
             })}
```

Comment:
I know this is pretty normal to see in Cloud, but I think we should start to do this when we intend to apply classes unconditionally. I'd like to hear your thoughts!

```suggestion
             className={classNames(classes.focusedRow, {
              [classes.disabledRow]:
                isSamePlan || planTooSmall || isDisabledClass,
            })}
```